templates/base/packages/nextjs/hooks/scaffold-eth/useScaffoldContractRead.ts

```diff
@@ -17,7 +20,7 @@ import {
  * @param config.functionName - name of the function to be called
  * @param config.args - args to be passed to the function call
  */
-export const useScaffoldContractRead = <
+export const useScaffoldReadContract = <
```

Comment:
This file should be removed, right??

We want to keep this one:
templates/base/packages/nextjs/hooks/scaffold-eth/useScaffoldReadContract.ts

Reply:
Ohh my bad sorry, got tricked by vsocde git UI or maybe I clicked wrong button while resolving conflicts of this file 

Removed this file and useScaffoldEventSubscriber too, now it nicely shows : 
![Screenshot 2024-04-16 at 4 44 23 PM](https://github.com/scaffold-eth/scaffold-eth-2/assets/80153681/88e737c0-0552-4f86-b15c-520c264ee236)


Tysm for noticing it ! 🙌